todo_scheduler.iml

```diff
@@ -1,18 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is there a specific reason for this file deletion?

Reply:
Oh I probably deleted that by accident when messing around with the library versions. I can add it back if it's necessary

Reply:
I looked more into it and the iml file can even be gitignored, because it will automatically generate, so no need to worry about re-adding it.